.github/workflows/cargo-test.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           override: true
       - run: cargo fetch --verbose
       - run: cargo build
+      - run: cargo build --package codechain-testmodule
```

Comment:
Why do we need this line?
`cargo build` does not build codechain-testmodule?

Reply:
No it doesn't since `codechain-testmodule` is not used anywhere.

Reply:
If the `cargo build` does not build codechain-testmodule since it is not used in the Foundry binary, how about trying `cargo build --workspace`?

>--workspace
    Build all members in the workspace.

You can find a help message of `cargo build` here: https://doc.rust-lang.org/cargo/commands/cargo-build.html

Reply:
Yes I've usually used this in local.
Then
```
- run: cargo build
- run: cargo build --package codechain-testmodule
```
to
```
- run: cargo build --workspace
```
?

Reply:
Yes, it looks better to me.